.github/workflows/ci.yml

```diff
@@ -660,6 +660,17 @@ jobs:
         echo "CPPINTEROP_DIR=$env:CPPINTEROP_DIR"
         echo "CPPINTEROP_DIR=$env:CPPINTEROP_DIR" >> $env:GITHUB_ENV  
 
+        #Build dlfcn-win32 so can use dlfcn.h on Windows
+        git clone --depth=1 https://github.com/dlfcn-win32/dlfcn-win32.git
```

Comment:
We should "just" use the relevant windows API. We do not need the dlfcn specifically.

Reply:
Ok. I will revert back to the old method. I only saw this comment after I put in the last commit.

Reply:
All windows builds are currently at the same stage now where errors such as missing header etc are stopping the build.

My attempts locally to work out how to get the LLVM_HASH and Cling_HASH without hardcoded have been unsuccessful

Reply:
I have reverted these changes now.